core/v2/validator.go

```diff
@@ -16,8 +16,8 @@ var (
 )
 
 type BlobShard struct {
-	BlobCertificate
-	Chunks map[core.QuorumID][]*encoding.Frame
+	*BlobCertificate
+	Chunks map[core.QuorumID]core.Bundle
```

Comment:
@mooselumph can you comment if you like this change?

Reply:
Looks fine to me!